### PR TITLE
Adds STW links to section index

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

https://transformuk.atlassian.net/browse/HOTT-941

### What?

I have added/removed/altered:

- [x] Added links to STW

### Why?

I am doing this because:

- This is required to get some traffic flowing to the STW project

### CODE REVIEW

**Comment by willfish** [comment]
image

File: app/webpacker/src/stylesheets/_feature-panel.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+$feature-panel-border-colour: #333;
+
+.feature-panel {
+  border-bottom: 1px solid $feature-panel-border-colour;
+  margin-bottom: 1em;
+  padding: 1em 1em 1em 0;
```
Comment (jebw) [inline app/webpacker/src/stylesheets/_feature-panel.scss]: There's some govuk mixins which can be used for padding? They change according to mobile/tablet/desktop ?

https://design-system.service.gov.uk/styles/spacing#spacing-on-custom-components

File: app/webpacker/src/stylesheets/_feature-panel.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+$feature-panel-border-colour: #333;
```
Comment (jebw) [inline app/webpacker/src/stylesheets/_feature-panel.scss:1]: I there are already variables for the govuk pallette - we should probably be using their variables for this?

https://design-system.service.gov.uk/styles/colour/

**Review by jebw — APPROVED** [review]
LGTM - the CSS could be integrating with styles provided by GovUK Design System ?

**Pull request merged**